manifests/init.pp

```diff
@@ -130,6 +130,10 @@
 #
 # $dhcp_subnets::               Subnets list to restrict DHCP management to
 #
+# $dhcp_ping_free_ip::          Perform ICMP and TCP ping when searching free IPs from the pool. This makes
```

Comment:
What's a TCP ping?

Reply:
Copied from https://github.com/theforeman/smart-proxy/blob/7c818d0cf1606a7d16c8334ab4fd256370eac365/config/settings.d/dhcp.yml.example#L14-L17. https://github.com/theforeman/smart-proxy/blob/7c818d0cf1606a7d16c8334ab4fd256370eac365/modules/dhcp_common/pingable.rb#L5-L40 suggests it's basically a TCP connect on port 7 (echo service).

Reply:
cool, thanks!